Resources/Prototypes/_Carpmosia/Entities/Mobs/NPCs/jorge.yml

```
@@ -0,0 +1,146 @@
+- type: entity
+  id: MobJorge
+  parent:
+  - BaseSimpleMob
+  - MobDamageable
+  - MobPolymorphable
+  - MobBloodstream
+  - MobFlammable
+  name: Jorge
+  description: A sentient potted plant originally mistaken for an anomaly. It's seen some shit.
+  suffix: Ghostrole
+  components:
+  - type: GhostRole
+    prob: 1
+    makeSentient: true
+    allowSpeech: true
+    name: ghost-role-information-jorge-name
+    description: ghost-role-information-jorge-description
+    rules: ghost-role-information-nonantagonist-rules
+    raffle:
+      settings: short
+  - type: GhostTakeoverAvailable
+  - type: CanEscapeInventory
+    baseResistTime: 3
+  - type: Item
+    size: Ginormous
+  - type: MultiHandedItem
+  - type: ComplexInteraction
+  - type: Hands
+    hands:
+      hand:
+        location: Left
+    sortedHands:
+    - hand
+  - type: Access
+    enabled: true
+    tags:
+    - Maintenance 
+    - Research
+  - type: Reactive
+    groups:
+      Flammable: [ Touch ]
+      Extinguish: [ Touch ]
+    reactions:
+    - reagents: [ Water ]
+      methods: [ Touch ]
+      effects:
+      - !type:HealthChange
+        damage:
+          groups:
+            Brute: -0.15
+    - reagents: [ Blood ]
+      methods: [ Touch ]
+      effects:
+      - !type:HealthChange
+        damage:
+          groups:
+            Brute: -0.25
+            Burn: -0.25
+    - reagents: [ RobustHarvest ]
+      methods: [ Touch ]
+      effects:
+      - !type:HealthChange
+        damage:
+          groups:
+            Brute: -1
+            Burn: -1
+    - reagents: [ WeedKiller, PlantBGone ]
+      methods: [ Touch ]
+      effects:
+      - !type:HealthChange
+        damage:
+          types:
+            Heat: 1
+  - type: BackwardsAccent
+  - type: Sprite
+    sprite: Structures/Furniture/potted_plants.rsi
+    noRot: true
+    state: plant-26
+  - type: Bloodstream
+    bloodReagent: Sap
+    bloodMaxVolume: 50
+    chemicalMaxVolume: 30
+  - type: Damageable
+    damageContainer: Biological
+    damageModifierSet: Diona
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTypeTrigger
+        damageType: Blunt
+        damage: 160
+      behaviors:
+      - !type:DoActsBehavior
+        acts: ["Destruction"]
+      - !type:PlaySoundBehavior
+        sound:
+          path: Audio/Effects/tree_fell.ogg
```

Weird idea, but try this?
```suggestion
          path: /Audio/Effects/tree_fell.ogg

---

It may also need to be in quotes, not sure